Dockerfile

```diff
@@ -1,6 +1,40 @@
-FROM node:10.13
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-COPY package.json yarn.lock ./
-RUN yarn --pure-lockfile
-COPY . .
+FROM node:10.13-alpine
+ARG COMMIT_HASH
+RUN test -n "$COMMIT_HASH"
+
+RUN apk add curl git
+
+# Set up deploy user and working directory
+RUN adduser -D -g '' deploy
+RUN mkdir -p /app
+RUN chown deploy:deploy /app
+
+# Set up dumb-init
```

Comment:
@izakp If I recall a previous conversation we had correctly, we'll want all/most Dockerfiles to contain this installation for dumb-init. I notice that we don't have a base node image in https://github.com/artsy/docker-images -- could it make sense to extract out the common-components of this image into a base image? Maybe could make future node imagines a bit more consistent.